rebootAtleatOnce.py

```diff
@@ -0,0 +1,13 @@
+import unittest
+from testutils import system
```

Comment:
We need to do relative import. We can use `from .testutils import system` instead.